content/10.diagnostics.md

```diff
@@ -154,6 +154,19 @@ On a related note, for some infectious agents an epidemic can be stopped from gr
 A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R0)) fraction of the population must be immune or, equivalently, less than (1/R0) fraction of the population susceptible [@isbn:9780199209996].
 However, for SARS-CoV-2 and COVID-19, because of the calculated R0 and mortality and the recorded per-region deaths and accumulated cases with the estimated factor of undetected cases, relying on herd immunity without vaccines and/or proven treatment options and/or strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.
 
+#### Sustained Immunity to COVID-19
+
+In the process of mounting a response to a viral pathogen, the immune system produces antibodies specific to the pathogen.
+Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior infections that are no longer active and to the development of vaccines.
+The two immunoglobulin classes most pertinent to these areas of interest are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most common antibodies found in the blood.
+Following SARS infection, IgM antibodies were first detected in most patients during the second week following infection and remained detectable until the twelfth week, while IgG antibodies were first detected in the second or third week and remained at close to their peak concentration through the thirteen weeks of the study [@doi:10.1056/NEJM200307313490520].
```

Comment:
Original reference here was a letter to editor. Potential replacement reference:

```suggestion
Following SARS infection, IgM and IgG antibodies were detected in the second week post-infection. IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180. IgG titers peaked by day 60, and persisted in all donors through the two-year duration of study. [@doi:10.1111/j.1440-1843.2006.00783.x].
```